tasking/serializers/location.py

```diff
@@ -47,8 +47,13 @@ def to_internal_value(self, value):
                 zip_file = zipfile.ZipFile(value.temporary_file_path())
             except AttributeError:
                 zip_file = zipfile.ZipFile(value)
+
             # Call get_shapefile method to get the .shp files name
-            shpfile = get_shapefile(zip_file)
+            try:
+                shpfile = get_shapefile(zip_file)
+            except Exception as exception:
```

Comment:
Would it be a good idea to catch only the specific exception(s) that you expect?